Gemfile.lock

```diff
@@ -266,125 +268,127 @@ GEM
       minitest (>= 5.1)
       tzinfo (~> 2.0)
       zeitwerk (~> 2.3)
-    addressable (2.8.4)
+    addressable (2.8.6)
       public_suffix (>= 2.0.2, < 6.0)
-    airbrake (11.0.1)
+    airbrake (11.0.3)
       airbrake-ruby (~> 5.1)
-    airbrake-ruby (5.2.0)
+    airbrake-ruby (5.2.1)
       rbtree3 (~> 0.5)
     ansible (0.2.2)
     ar_multi_threaded_transactional_tests (0.5.0)
       activerecord (>= 4.2.0, < 6.2.0)
-    ast (2.4.1)
+    ast (2.4.2)
     attr_encrypted (3.1.0)
       encryptor (~> 3.0.0)
-    audited (5.4.2)
+    audited (5.4.3)
       activerecord (>= 5.0, < 7.2)
       request_store (~> 1.2)
-    autoprefixer-rails (9.4.8)
+    autoprefixer-rails (9.8.6.5)
       execjs
-    awesome_print (1.6.1)
-    aws-eventstream (1.2.0)
-    aws-partitions (1.329.0)
-    aws-sdk-core (3.100.0)
-      aws-eventstream (~> 1, >= 1.0.2)
-      aws-partitions (~> 1, >= 1.239.0)
+    awesome_print (1.9.2)
+    aws-eventstream (1.3.0)
+    aws-partitions (1.887.0)
+    aws-sdk-core (3.191.0)
+      aws-eventstream (~> 1, >= 1.3.0)
+      aws-partitions (~> 1, >= 1.651.0)
+      aws-sigv4 (~> 1.8)
+      jmespath (~> 1, >= 1.6.1)
+    aws-sdk-ecr (1.69.0)
+      aws-sdk-core (~> 3, >= 3.191.0)
       aws-sigv4 (~> 1.1)
-      jmespath (~> 1.0)
-    aws-sdk-ecr (1.3.0)
-      aws-sdk-core (~> 3)
-      aws-sigv4 (~> 1.0)
-    aws-sdk-kms (1.23.0)
-      aws-sdk-core (~> 3, >= 3.58.0)
+    aws-sdk-kms (1.77.0)
+      aws-sdk-core (~> 3, >= 3.191.0)
       aws-sigv4 (~> 1.1)
-    aws-sdk-s3 (1.45.0)
-      aws-sdk-core (~> 3, >= 3.58.0)
+    aws-sdk-s3 (1.143.0)
+      aws-sdk-core (~> 3, >= 3.191.0)
       aws-sdk-kms (~> 1)
-      aws-sigv4 (~> 1.1)
-    aws-sigv4 (1.6.1)
+      aws-sigv4 (~> 1.8)
+    aws-sigv4 (1.8.0)
       aws-eventstream (~> 1, >= 1.0.2)
-    base64 (0.1.1)
+    base64 (0.2.0)
+    bigdecimal (3.1.6)
     binding_of_caller (0.8.0)
       debug_inspector (>= 0.0.1)
-    bootsnap (1.4.5)
-      msgpack (~> 1.0)
+    bootsnap (1.18.3)
+      msgpack (~> 1.2)
     bootstrap-sass (3.4.1)
       autoprefixer-rails (>= 5.2.1)
       sassc (>= 2.0.0)
     bootstrap3-datetimepicker-rails (4.17.47)
       momentjs-rails (>= 2.8.1)
     brakeman (5.4.1)
     builder (3.2.4)
-    bundler-audit (0.9.0.1)
+    bundler-audit (0.9.1)
       bundler (>= 1.2.0, < 3)
       thor (~> 1.0)
     byebug (8.2.5)
-    chef-utils (18.1.29)
+    chef-utils (18.4.2)
```

Comment:
### [chef-utils](https://github.com/chef/chef/tree/main/chef-utils)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 18.1.29 → 18.4.2

[_[change-log](https://github.com/chef/chef/blob/main/CHANGELOG.md), [source-code](https://github.com/chef/chef/tree/main/chef-utils), [gem-diff](https://my.diffend.io/gems/chef-utils/18.1.29/18.4.2)_]





<details>
<summary>Commits</summary>

A change of **403** commits. See the full changes on [the compare page](https://github.com/chef/chef/compare/v18.1.29...v18.4.2).

These are the first 10 commits:
- (448d438) [Bump version to 18.2.31 by Chef Expeditor](https://github.com/chef/chef/commit/448d438e645b00677b4c2197d1446bc508f9baec)
- (b3cbd76) [Update for chef-foundation ver to pick openssl-1.0.2ze](https://github.com/chef/chef/commit/b3cbd76ac1609dd9a488aefc4d4e747154f0c45d)
- (8576ef3) [remove openssl-src](https://github.com/chef/chef/commit/8576ef3d33562e80991b6c45329cb338d79685a5)
- (72fd888) [Adding double-quotes around path for Windows 'secedit' comma…](https://github.com/chef/chef/commit/72fd88829b3ca16a49ef39af22dcc0bc6c64e10d)
- (c89c12b) [Make action homebrew_bin_path public which is used in other …](https://github.com/chef/chef/commit/c89c12b0d14978f9957c6f39b50ef843eea94ed9)
- (1e7a3bb) [Bump version to 18.2.32 by Chef Expeditor](https://github.com/chef/chef/commit/1e7a3bb5728963b135f7699356def6e8fcce00fe)
- (01c23ea) [Optimize Chef::CookbookSynchronizer#remove_deleted_files (#1…](https://github.com/chef/chef/commit/01c23ea73487dc9b6ee86edc7b797c12e3b1a14a)
- (3d7c78e) [Bump version to 18.2.33 by Chef Expeditor](https://github.com/chef/chef/commit/3d7c78e8afa1ef54e8ded8e914daf727607b3fb3)
- (5ab62ef) [update ::File.exists? to ::File.exist?](https://github.com/chef/chef/commit/5ab62efe29f632dd5d5f22f90db0f352f43cdf94)
- (be390a7) [Merge pull request #13807 from chef/poorndm/test-openssl-1.0…](https://github.com/chef/chef/commit/be390a72a055375bca4ea1861c57a1ce99da0710)


</details>